lib/dotiw/action_view/helpers/date_helper.rb

```diff
@@ -37,6 +29,17 @@ def distance_of_time_in_percent(from_time, current_time, to_time, options = {})
         result = ((current_time - from_time) / distance) * 100
         number_with_precision(result, options).to_s + '%'
       end
+
+      private
+      def merge_options(include_seconds_or_options, options)
+        merged_options = options.dup
```

Comment:
This seems like it's doing more work than needed in all cases, duping and merging. This is simpler:

```ruby
def merge_options(include_seconds_or_options, options)
  if include_seconds_or_options.is_a?(Hash)
    options.merge(include_seconds_or_options)
  else
    options.merge(include_seconds: !!include_seconds_or_options)
  end
end
```

Reply:
Sorry, newbie error, I thought you were making the change, not waiting for me to do it.
Change as suggested